src/models/userModel.ts

```diff
@@ -9,7 +9,7 @@ const urserSchema = new Schema<IUser>(
     lastName: { type: String, required: true },
     gender: {
       type: String,
-      enum: ["Male", "Female", "Neither"],
+      enum: ["male", "female", "neither"],
```

Comment:
This can go into an enum interface and user here.
```suggestion
      enum: Object.values(GenderEnum),
```